app/code/Magento/Checkout/view/frontend/web/js/view/payment/default.js

```diff
@@ -18,8 +18,8 @@ define([
     'Magento_Checkout/js/model/payment/additional-validators',
```

Comment:
Please, revert all of these changes, the code style check will fail. https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-javascript.html?itm_source=devdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=eslin